### DISCUSSION TITLE
Where do cron logs go?

### DISCUSSION THREAD

**Discussion by jfmatth** [dokku/dokku#4986] [Q&A]
Hello, thanks again for Dokku, great project!

I'm trying to move to Dokku, but can't find a log of my cron jobs.  I don't know if they are running or not.  From everything I check, it looks like they are in the crontab, the testing i do manually seems to be OK, but I would like to see logs.

thanks,

**Comment by vnorbix** [comment]
I don't know if you're referring to the application level cron jobs in `app.json`. In this case Dokku cron jobs are registered in your host system's cron service as the dokku user. Ubuntu based system you can check this with command `sudo -u dokku crontab -e`, dokku run command is used to execute the command you specify in app.json. To see the output of the job execution you can use command `sudo grep CRON /var/log/syslog`. Here I got `No MTA installed, discarding output` message, which suggested me to just redirect all the output (stdout and stderr) to a file on the host. Something like this would do:

```
mycommand &>> /var/log/myservice/somelog.txt
```

The `myservice` directory here need to be writeable by the dokku user.

**Reply by cba85** [reply]
Thank you, your reply helped me too!

I've added postfix to log my Dokku crons. I've used this tutorial: https://cronitor.io/guides/no-mta-installed-discarding-output

Hope it would help someone else!
